go/coinstacks/cosmos/api/handlers.go

```diff
@@ -69,9 +69,10 @@ func (h *Handler) GetAccount(pubkey string) (api.Account, error) {
 	return account, nil
 }
 
-func (h *Handler) GetTxHistory(pubkey string, page int, pageSize int) (api.TxHistory, error) {
-	res, err := h.httpClient.GetTxHistory(pubkey, page, pageSize)
+func (h *Handler) GetTxHistory(pubkey string, cursor string, pageSize uint) (api.TxHistory, error) {
+	res, err := h.httpClient.GetTxHistory(pubkey, cursor, pageSize)
 	if err != nil {
+		logger.Errorf("error getting tx history for %s (%s): %s", pubkey, cursor, err)
```

Comment:
Prefer wrapping errors with annotations on only logging at the top level caller

Reply:
believe I added this line to try to get at swallowed errors from the issues with the openapi generated client.